loopchain/blockchain/blockchain.py

```diff
@@ -1442,3 +1442,8 @@ def makeup_block(self,
             self.__add_tx_to_block(block_builder)
 
         return block_builder
+
+    def try_update_last_unconfirmed_block(self, unconfirmed_block: 'Block'):
+        """Note that the method expects an input as Block 1.0+"""
+        if not self.__last_block or unconfirmed_block.header.height == self.__last_block.header.height + 2:
```

Comment:
If this condition check is to determine a normal leader, shouldn't the effect of this check be reflected in send_unconfirmed_block(of runner.py) as well? If not, wouldn't it be okay to not have this condition check?